### PR TITLE
EZP-25111: Deprecated ViewController location actions

### PR DESCRIPTION
**Pull request by bdunogier**
> http://jira.ez.no/browse/EZP-25111

### CODE REVIEW

**Comment by andrerom** [comment]
+1 with cs fixes

File: eZ/Publish/Core/MVC/Symfony/Controller/Content/ViewController.php [hunk 1]
```diff
@@ -144,6 +144,12 @@ protected function buildResponse($etag = null, DateTime $lastModified = null)
      */
     public function viewLocation($locationId, $viewType, $layout = false, array $params = array())
     {
+        trigger_error(
+            "ViewController:viewLocation() is deprecated since kernel 6.0.0, and will be removed in the future.\n" .
```
Comment (glye) [inline eZ/Publish/Core/MVC/Symfony/Controller/Content/ViewController.php]: Should be double colon, no? Same for the others.

**Comment by andrerom** [comment]
fixed cs, merge on pass If I have not.

**Comment by bdunogier** [comment]
Thank you guys.

**Pull request merged**